gradle-conjure/src/main/java/com/palantir/gradle/conjure/ConjureGeneratorTask.java

```diff
@@ -87,27 +92,29 @@ protected File outputDirectoryFor(File _file) {
 
     /** Entry point for the task. */
     public void compileFiles() {
-        getSource().getFiles().forEach(file -> {
-            File thisOutputDirectory = outputDirectoryFor(file);
-
+        Map<String, String> environment = System.getenv();
+        WorkQueue workQueue = getWorkerExecutor().processIsolation(processWorkerSpec -> {
```

Comment:
I need to read up exactly what this does - but if you know already, does this spin up one process per source file or one process for **all** the source files (that is not in the gradle daemon?)

Reply:
https://docs.gradle.org/7.4/userguide/worker_api.html#creating_a_worker_daemon implies to me this will create one worker daemon for all the tasks.

Reply:
yep that was my understanding!